zadanie_8_6/zadanie_8_6.js

```diff
@@ -0,0 +1,20 @@
+var lenght = prompt('Podaj długość boku: '), 
+	height = prompt('Podaj wysokość: ');
+
+function getTriangleArea(a, h) {
+	if ((a > 0) && (h > 0)) {
+		return (a*h/2);
+	} else {
+		return 'Nieprawidłowe dane';
+	}
+}
+
+document.write('Pole Twojego trójkąta: ' + getTriangleArea(lenght, height));
+
+var triangle1Area = getTriangleArea(1, 5);
+var triangle2Area = getTriangleArea(7, 3);
+var triangle3Area = getTriangleArea(1434, 5232);
+
+document.write(' Pole 1: ' + triangle1Area);
+document.write(' Pole 2: ' + triangle2Area);
+document.write(' Pole 3: ' + triangle3Area);
```

Comment:
jesli uzywasz document.write to zadbaj tez o czytelnosc - obecnie wszystko jest w jednej linii